### DISCUSSION TITLE
Please send the UI/UX Video  mentioned in Community call

### DISCUSSION THREAD

**Discussion by khalidk799** [EddieHubCommunity/support#1002] [Community Call]
Hey Eddie, Please send the video you were talking about in yesterday's community talk. It was about  a Ui/Ux design you saw at a hackthon. Which according to you we won't get any justice unless we see the video.  :P

**Comment by eddiejaoude** [comment]
I can't find the video, but here is a photo

![ux](https://cdn.discordapp.com/attachments/800436994849112134/815661094404096020/IMG_0378.JPG)

**Reply by khalidk799** [reply]
Thanks @eddiejaoude
